python/pyspark/ml/feature.py

```diff
@@ -120,6 +122,200 @@ def getThreshold(self):
         return self.getOrDefault(self.threshold)
 
 
+class LSHParams(Params):
```

Comment:
The classes in this file are alphabetized for the most part. Let's keep the convention here.